### PR TITLE
[JS] Topic: Async_and_Await

### LINKED ISSUE #721: JavaScript: What is Async and Await?
Description: I Will be explaining what is Async and Await in JavaScript (ES6).

The async keyword turns a method into an async method, which allows you to use the await keyword in its body. When the await keyword is applied, it suspends the calling method and yields control back to its caller until the awaited task is complete. await can only be used inside an async method.
Changes will be inside winter-of-contributing/Web_Development/Javascript/Topics/ branch.

I am a GWOC'21 contributor from Batch 1 of JavaScript. Please assign this issue to me. I assure you that this issue's title will be the same as my PR's title as well as I will be Contributing under the Guidelines and Code of Conduct.

### PR DESCRIPTION
**Pull request by 7sakshi7**
<hr>

## Description 📜

Created a tutorial on Aysnc and Await in Javascript and elaborated upon the following:

What is Async functions?
What is Await functions?
Error Handling 
Multiple await function

Resolves #721

<hr>

## Type of change 📝

- [ ] Au#dio (Should be in mp3 format Includes speech clarity, Concise ,Low distortion)
- [ ] Vi#deo (Animations, screen-recordings, presentations and regular explanatory films are all possibilties etc)
- [x] Doc#umentation (Content Creation in the form of codes or tutorials)
- [ ] Other (If you choose other, Please mention changes below) 

<hr>

## Domain of Contribution 📊

- [ ] Android Dev #(Flutter)
- [ ] Android Dev #(Java)
- [ ] Android Dev #(Kotlin)
- [ ] Backend Dev #(Java)
- [ ] Backend Dev #(.NET)
- [ ] Backend Dev #(PHP)
- [ ] Backend Dev #(Python)
- [ ] C/#CPP
- [ ] Competitive #Programming
- [ ] Cyber #Security
- [ ] DS#A
- [ ] Data#base
- [ ] Datascience with #Python
- [ ] Datascience with #R
- [ ] Frontend Dev #HTML/CSS/JS
- [ ] Frontend Dev #React/Angular/Vue
- [ ] Go#lang
- [ ] Interview #Prep
- [ ] Java_#Domain
- [x] Java#Script
- [ ] ME#RN
- [ ] Machine #Learning
- [ ] Open #Source
- [ ] Python_#Domain
- [ ] Ru#st
- [ ] Statis#tics
- [ ] UI/#UX

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by dakshsethi** [comment]
@7sakshi7 First of all, try to have less number of commits before using `/assign`.

**Comment by paramsiddharth** [comment]
@Creatoon @dakshsethi @s4-2 Please review.

File: JavaScript/Topics/Aync And Await/README.md
Comment (paramsiddharth) [inline JavaScript/Topics/Aync And Await/README.md:12]:
```suggestion
````js
```

This will enable syntax highlighting. Do this for all code blocks.

**Review by paramsiddharth — CHANGES_REQUESTED** [review]

File: JavaScript/Topics/Aync And Await/README.md
Comment (7sakshi7) [reply JavaScript/Topics/Aync And Await/README.md:14]: @paramsiddharth I have done the changes, Kindly review it.

File: JavaScript/Topics/Aync And Await/README.md
Comment (dakshsethi) [inline JavaScript/Topics/Aync And Await/README.md:14]: use ```js everywhere you have js code snippets.

**Review by dakshsethi — CHANGES_REQUESTED** [review]

File: JavaScript/Topics/Aync And Await/README.md
Comment (7sakshi7) [reply JavaScript/Topics/Aync And Await/README.md]: Done

**Review by dakshsethi — APPROVED** [review]
Looks good to me!! 🔥🔥🔥

File: JavaScript/Topics/Aync And Await/README.md
Comment (paramsiddharth) [inline JavaScript/Topics/Aync And Await/README.md:154]: Please add credits to yourself in the following manner:

```md
Contributor: [Sakshi Agarwal](https://github.com/7sakshi7)
```

**Review by paramsiddharth — CHANGES_REQUESTED** [review]
One more change and you're good to merge after another approval.

File: JavaScript/Topics/Aync And Await/README.md
Comment (7sakshi7) [reply JavaScript/Topics/Aync And Await/README.md:156]: Added Sir. Kindly review and if at all there is any changes to be done , let me know.

Comment (7sakshi7) [reply JavaScript/Topics/Aync And Await/README.md:156]: @paramsiddharth Kindly review my PR. So that it can be merged.

**Comment by paramsiddharth** [comment]
@7sakshi7 Make sure you attend our acknowledgement session at 7 PM tomorrow. Good luck!

File: JavaScript/Topics/Aync And Await/README.md
Comment (s4-2) [inline JavaScript/Topics/Aync And Await/README.md:9]: **An** Async function is a function declared with **the**  **Async** keyword **which** means it will run in the **background** while performing the code inside when the function is done, it automatically returns a promise.
@7sakshi7 Please correct some grammar and spelling errors and after that, I think you are good to go. Cheers!

Comment (7sakshi7) [reply JavaScript/Topics/Aync And Await/README.md]: @s4-2 Done

**Comment by paramsiddharth** [comment]
@Creatoon @s4-2 @amino19 Please review.

File: JavaScript/Topics/Aync And Await/README.md
Comment (amino19) [inline JavaScript/Topics/Aync And Await/README.md:1]:
```suggestion
```

File: JavaScript/Topics/Aync And Await/README.md
Comment (amino19) [inline JavaScript/Topics/Aync And Await/README.md:46]:
```suggestion
Output-
```

File: JavaScript/Topics/Aync And Await/README.md
Comment (amino19) [inline JavaScript/Topics/Aync And Await/README.md:82]:
```suggestion
Output-
```

**Review by amino19 — CHANGES_REQUESTED** [review]
@7sakshi7: Thanks for your contribution, some minor changes reg. "missing review" :eyes:
Also, pl check each & every sentence simultaneously, maybe I missed something. :sweat_smile:

File: JavaScript/Topics/Aync And Await/README.md
Comment (7sakshi7) [reply JavaScript/Topics/Aync And Await/README.md]: @amino19 I have done all the changes. You can check

**Review by s4-2 — APPROVED** [review]
Well done @7sakshi7. Looks good to me:)

**Review by amino19 — APPROVED** [review]
@7sakshi7: LGTM! :partying_face:

File: JavaScript/Topics/Aync And Await/README.md
Comment (amino19) [reply JavaScript/Topics/Aync And Await/README.md]: @7sakshi7: deffo, you fixed it somehow, yayy! :balloon:

**Pull request merged**
